src/main/java/com/spotinst/sdkjava/model/SpotinstElastigroupClient.java

```diff
@@ -676,4 +676,127 @@ public Boolean removeSuspendedScalingPolicies(
         }
         return retVal;
     }
-}
+
+    public ElastigroupStartDeploymentResponse startDeployment(ElastigroupStartDeploymentRequest startDeploymentRequest, String elastigroupId) {
+
+        ElastigroupStartDeploymentResponse isDeploymentStarted;
+
+        ElastigroupStartDeployment elastigroupStartDeploymentRequest = startDeploymentRequest.getElastigroupDeployment();
```

Comment:
Didn't understand this line.

Reply:
It is used to get Request Body for startDeloyment.